### PR TITLE
[17.0][MIG] report_qweb_pdf_watermark

### PR DESCRIPTION
**Pull request by sanderlienaerts**

### CODE REVIEW

**Review by david-s73 — APPROVED** [review]
Functional review okay

**Comment by sanderlienaerts** [comment]
@gurneyalex could you please look into this PR?

File: report_qweb_pdf_watermark/README.rst [hunk 1]
```diff
@@ -38,103 +42,106 @@ This module was written to add watermarks (backgrounds) to PDF reports. Because
...
+1. go to settings --> company --> update info
+2. upload an pdf watermark
+3. go to settings --> technical --> reporting --> reports
+4. Select the report where you want to use it.
+5. On the 'Advanced Properties' tab of the notebook check 'use company
+   watermark'
 
-### Demo
-And demo report is available (if you have demo data installed) on the users form view.
+### Demo And demo report is available (if you have demo data installed)
```
Comment (Larsq1) [inline report_qweb_pdf_watermark/README.rst:71]: Check this

File: report_qweb_pdf_watermark/static/description/index.html [hunk 2]
```diff
@@ -406,19 +410,22 @@ <h1><a class="toc-backref" href="#toc-entry-2">Usage</a></h1>
...
 <li>upload an pdf watermark</li>
 <li>go to settings –&gt; technical –&gt; reporting –&gt; reports</li>
 <li>Select the report where you want to use it.</li>
-<li>On the ‘Advanced Properties’ tab of the notebook check ‘use company watermark’</li>
+<li>On the ‘Advanced Properties’ tab of the notebook check ‘use company
+watermark’</li>
 </ol>
-<p>### Demo
-And demo report is available (if you have demo data installed) on the users form view.</p>
+<p>### Demo And demo report is available (if you have demo data installed)
```
Comment (Larsq1) [inline report_qweb_pdf_watermark/static/description/index.html:428]: Check this

**Review by Larsq1 — APPROVED** [review]
Looks good to me

**Comment by Larsq1** [comment]
@sanderlienaerts please clean up the commit history

**Comment by HviorForgeFlow** [comment]
@sanderlienaerts can you attend this comment? Thanks for contributing

https://github.com/OCA/reporting-engine/pull/841#issuecomment-2311772855

**Comment by HviorForgeFlow** [comment]
ping @sanderlienaerts

**Review by bosd — APPROVED** [review]

**Review by CasVissers-360ERP — APPROVED** [review]
Functional review

**Pull request merged**